docs/learn/encyclopedia/contract-development/stellar-contract.mdx

```diff
@@ -0,0 +1,141 @@
+---
+sidebar_position: 10
+title: Introduction
+description: Introduction to Smart Contracts
+---
+
+<head>
+  <title>{`The introduction to smart contracts.`}</title>
```

Comment:
now that the whole `<title>...</title>` thing is on one line, you no longer _need_ the brackety-backticky, ``{`...`}``, part of it (confusing, i know! lol). those are only _required_ when the text inside the `<title>` tag is on its own line or spans multiple lines. that said, it's not gonna harm anything to have them here.